dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
@terrywbrady : I think minimally this needs a comment to its purpose. It's really odd looking in a test, as it implies that Groups can exist without a name (and the test now supports that concept, when it probably shouldn't).  My original comment was stating that we may want to look closer at the *other* tests in this class to see which ones are creating groups that have no name, as they may be more "at fault" here.  If we can fix the other tests, then this check for null names should be unnecessary.

As an example the `createGroup()` test here creates an unnamed Group. But, it does a good job of cleaning up that unnamed group in a `finally` section: https://github.com/Georgetown-University-Libraries/DSpace/blob/ebf256caa1f03194cb547dae7eada64428a1b3e2/dspace-api/src/test/java/org/dspace/eperson/GroupTest.java#L121    I'm wondering if we have one or more other test cases here that are creating unnamed groups but *not* cleaning them up in a `finally` section.... as those may be the root cause of your issue here.